crates/floresta-chain/src/pruned_utreexo/chainparams.rs

```diff
@@ -20,6 +20,8 @@ use crate::Network;
 pub struct ChainParams {
     /// The network's first block, also called genesis block.
     pub genesis: Block,
+    /// Wheter or not to verify script. Requires "bitcoinconsensus" feature.
+    pub verify_script: bool,
```

Comment:
ok

Reply:
IIRC the decision whether to verify or not scripts comes from a method that just checks whether we are past the `assume_valid` index. Partial chains have this as a field because we can take intervals before and after the `assume_valid`.